<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Cascadia Code</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ReVox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202020;
color: #DDDDDD;
font-family: &quot;Cascadia Code&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>80</width>
      <height>509</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #101010;
border: 0px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>430</y>
       <width>82</width>
       <height>80</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border: 1px solid #202020;
}

QToolButton:pressed {
	background-color: #252525;
}

QToolButton:hover {
	background-color: #151515;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../appicon.qrc">
       <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="soundboardButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>82</width>
       <height>80</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Soundboard&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border: 1px solid #202020;
}

QToolButton:pressed {
	background-color: #252525;
}

QToolButton:hover {
	background-color: #151515;
}

QToolButton:disabled{
	background-color: #202020;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../appicon.qrc">
       <normaloff>:/icons/soundboard.png</normaloff>:/icons/soundboard.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="fxButton">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>80</y>
       <width>82</width>
       <height>80</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Voice FX&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
	border: 1px solid #202020;
}

QToolButton:pressed {
	background-color: #252525;
}

QToolButton:hover {
	background-color: #151515;
}

QToolButton:disabled{
	background-color: #202020;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../appicon.qrc">
       <normaloff>:/icons/fx.png</normaloff>:/icons/fx.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionDevice_Configuration">
   <property name="text">
    <string>Device Configuration</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
  <action name="actionDevice_Setup">
   <property name="text">
    <string>Device Setup</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../appicon.qrc"/>
 </resources>
 <connections/>
</ui>
